spec.bs

```diff
@@ -245,6 +245,77 @@ Run the [=update document frozenness steps=] given <var ignore>child document</v
 
 Each {{HTMLMediaElement}} has a <dfn for="HTMLMediaElement">resume frozen flag</dfn>, which is initially set to false.
 
+Modifications to the Service Worker Standard {#serviceworker-mod}
+--------------------------------------------
+
+### Service Worker: <a href="https://w3c.github.io/ServiceWorker/#client-interface">Client Interface</a> ### {#serviceworker-client-dfn}
+
+<pre class="idl">
+    partial interface Client {
+        readonly attribute  ClientState state;
+    };
+
+    enum ClientState {
+        "active",
+        "frozen"
+    };
+</pre>
+
+A {{Client}} object has an associated <dfn id="dfn-service-worker-client-state" for="Client">state</dfn>, which is one of {{ClientState}} attribute value.
```

Comment:
Nothing ever sets this value. That seems bad.

Reply:
I skipped adjusting the Create Client algorithms where state is passed in.. I've now added them so it should be clearer.

Reply:
Thanks, this is clearer. The remaining thing is to add two subsections of "Algorithms" that explains how you modify "Create Window Client" and "Create Client" to take the new argument.

Reply:
Done